src/main/java/Duke.java

```diff
@@ -1,10 +1,193 @@
+                    System.out.println("Congratulations! You have completed: ");
+                    System.out.println((taskNumInt + 1) + ". " + entireList[taskNumInt].toString());
+                    System.out.println(DASH_LINE);
+                }
+                //Invalid "done" command
+                else {
+                    System.out.println(DASH_LINE);
+                    System.out.println("Errr... I don't think you have this item on the list.");
+                    System.out.println("This is the current list that you have:");
+                    printEntireList(entireList);
+                    System.out.println(DASH_LINE);
+                }
+            }
+
+            else if(userCommand.startsWith("deadline")){
+
+                //Valid "deadline" command
+                if(userCommand.contains("/by")){
+                    int dividerPosition = userCommand.indexOf("/by");
+                    String userCommandName = userCommand.substring(LENGTH_OF_DEADLINE+1, dividerPosition).trim();
+                    String userCommandBy = userCommand.substring(dividerPosition + LENGTH_OF_BY+1).trim();
+
+                    Deadline d = new Deadline(userCommandName, userCommandBy);
+                    entireList[counterList] = d;
+                    counterList++;
+
+                    printReplyForAddCommand(counterList, DASH_LINE, d.toString());
+                }
+
+                //Invalid "deadline" command
+                else {
+                    isInvalid = true;
+                }
+            }
+
+            else if(userCommand.startsWith("event")) {
+
+                //Valid "Events" command
+                if(userCommand.contains("/at")){
+                    int dividerPosition = userCommand.indexOf("/at");
+                    String userCommandName = userCommand.substring(LENGTH_OF_EVENT+1, dividerPosition).trim();
+                    String userCommandBy = userCommand.substring(dividerPosition + LENGTH_OF_BY+1).trim();
+
+                    Events e = new Events(userCommandName, userCommandBy);
+                    entireList[counterList] = e;
+                    counterList++;
+
+                    printReplyForAddCommand(counterList, DASH_LINE, e.toString());
+                }
+
+                //Invalid "Events" command
+                else {
+                    isInvalid = true;
+                }
+            }
+
+            else if(userCommand.startsWith("todo")) {
+                String userCommandName = userCommand.substring(LENGTH_OF_TODO+1).trim();
+
+                ToDos t = new ToDos(userCommandName);
+                entireList[counterList] = t;
+                counterList++;
+
+                printReplyForAddCommand(counterList, DASH_LINE, t.toString());
+            }
+
+            else if(userCommand.equals("?")) {
+                isUnsure = true;
+            }
+
+            else {
+                isUnsure = true;
+            }
```

Comment:
if-else should have the following form

if (condition) {
    statements;
} else if (condition) {
    statements;
} else {
   statements;
}

Reply:
Okie, I will take note of this. Thank you!